Source/Neptune.Web/Models/WaterQualityManagementPlan.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Web;
+using LtInfo.Common;
+using Neptune.Web.Common;
+using Neptune.Web.Controllers;
+
+namespace Neptune.Web.Models
+{
+    public partial class WaterQualityManagementPlan : IAuditableEntity
+    {
+        public string AuditDescriptionString => $"Water Quality Management Plan \"{WaterQualityManagementPlanName}\"";
+
+        public string GetDetailUrl() => SitkaRoute<WaterQualityManagementPlanController>.BuildUrlFromExpression(c =>
```

Comment:
I'm used to seeing the Get*Url() methods as extension methods (e.g. in a static WaterQualityManagementPlanModelExtensions class), but I think I like this better.

Reply:
Either one works and static might be a more performant option. I'll revisit this and probably use a static template, but I probably wont move it to the Model Extension.